pep/app/pods/components/word-wheel/component.ts

```diff
@@ -0,0 +1,69 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import DS from 'ember-data';
+import { removeEmptyQueryParams } from '@gavant/ember-pagination/utils/query-params';
+
+import { FlTypeaheadSuggestion } from 'pep/pods/components/fl-typeahead/component';
+
+interface WordWheelArgs {
+    placeholder: string;
+    value?: string;
+    limit?: number;
+    apiField?: string;
+    apiCore?: string;
+    onChange: (newText: string) => void;
+}
+
+export default class WordWheel extends Component<WordWheelArgs> {
+    @service store!: DS.Store;
+
+    @tracked suggestions: FlTypeaheadSuggestion[];
```

Comment:
Im curious as to why you don't just default them here and remove the constructor

```
@tracked suggestions: FlTypeaheadSuggestion[] = [];
```

Reply:
old habits from dealing with the dreaded `Ember.Object`-based classes' "shared state" gotcha 😛, where object/array property values would be shared across all instances of the component if you defaulted them this way. 

But yea this should be ok now with native classes, so I can clean this up.